### PR TITLE
Kleborate updates

### PR DESCRIPTION
**Pull request by kapsakcj**
~~Draft for now, until I'm done testing in Terra (just need to test Single End workflow)~~

Done to address issue #142 

Updates regarding Kleborate:

- default docker image updated to `quay.io/staphb/kleborate:2.2.0` from 2.0.4
- 4 new outputs from kleborate task (column header in Terra):
  - `K_TYPE`    (`kleborate_ktype`)
  - `O_TYPE`    (`kleborate_otype`)
  - `K_LOCUS_CONFIDENCE`    (`kleborate_klocus_confidence`)
  - `O_LOCUS_CONFIDENCE`    (`kleborate_olocus_confidence`)
- added 4 new outputs to both TheiaProk_Illumina_pe and theiaprok_illumina_SE workflows
- added 4 new outputs to export_taxon_tables task

PE workflow tested succesfully in Terra, 4 new outputs also exported to new data table ✅ 

SE workflow undergoing testing now in Terra

~~Will likely need to update CI where necessary. Expect it to fail on first attempt~~ OK, I was wrong. CI passed 🎉

### CODE REVIEW

**Comment by kapsakcj** [comment]
SE testing - test submission here: https://app.terra.bio/#workspaces/cdph-terrabio-taborda-manual/curtis_sandbox/job_history/7b32faa7-a8fc-4166-a557-c8c776e78714

Need to look at Kleborate output columns and double check export_taxon_tables ran as expected too (see `amr_specimen` table)

**Comment by kapsakcj** [comment]
Single End workflow ran as expected too ✔️ 

Ready for review & testing

File: tasks/utilities/task_broad_terra_tools.wdl [hunk 1]
```diff
@@ -175,9 +179,9 @@ task export_taxon_tables {
...
...ulence_report}\t~{amrfinderplus_version}\t~{amrfinderplus_db_version}\t~{amrfinderplus_amr_genes}\t~{amrfinderplus_stress_genes}\t~{amrfinderplus_virulence_genes}\t~{genotyphi_report_tsv}\t~{genotyphi_mykrobe_json}\t~{genotyphi_version}\t~{genotyphi_species}\t~{genotyphi_st_probes_percent_coverage}\t~{genotyphi_final_genotype}\t~{genotyphi_genotype_confidence}\t~{ani_highest_percent}\t~{ani_highest_percent_bases_aligned}\t~{ani_output_tsv}\t~{ani_top_species_match}\t~{ani_mummer_version}\t~{resfinder_pheno_table}\t~{resfinder_pheno_table_species}\t~{resfinder_seqs}\t~{resfinder_results}\t~{resfinder_pointfinder_pheno_table}\t~{resfinder_pointfinder_results}\t~{resfinder_db_version}\t~{resfinder_docker}\t~{prokka_gff}\t~{prokka_gbk}\t~{prokka_sqn}\t~{plasmidfinder_plasmids}\t~{plasmidfinder_results}\t~{plasmidfinder_seqs}\t~{plasmidfinder_docker}\t~{plasmidfinder_db_version}\t~{pbptyper_predicted_1A_2B_2X}\t~{pbptyper_pbptype_predicted_tsv}\t~{pbptyper_version}\t~{poppunk_gps_cluster}\t~{poppunk_gps_external_cluster_csv}\t~{poppunk_version}\t~{seroba_version}\t~{seroba_serotype}\t~{seroba_ariba_serotype}\t~{seroba_ariba_identity}\t~{seroba_details}"  >> ~{samplename}_terra_table.tsv
```
Comment (frankambrosio3) [inline tasks/utilities/task_broad_terra_tools.wdl:184]: Just a formatting comment, there is a space before the "~" in {kleborate_klocus_confidence}\t ~

**Review by frankambrosio3 — APPROVED** [review]
Looks good, runs successfully on Terra

**Comment by kapsakcj** [comment]
after dicussion with @emmadoughty I think we'll end up adding columns for the `k_locus` column from the output kleborate TSV

File: tasks/utilities/task_broad_terra_tools.wdl (hunk 1 above)
Comment (kapsakcj) [reply tasks/utilities/task_broad_terra_tools.wdl:184]: good catch, I will fix this. Luckily this didn't break things

Comment (kapsakcj) [reply tasks/utilities/task_broad_terra_tools.wdl:184]: I found a few more of these 🤭

**Comment by kapsakcj** [comment]
Added one final output column `kleborate_docker` in [bc861b3](https://github.com/theiagen/public_health_bacterial_genomics/pull/148/commits/bc861b391bdfae159ffb30e305f5dd6222026911)

Tested fine in Terra for both PE and SE workflows

This PR is ready for testing and review

**Review by emmadoughty — APPROVED** [review]
This looks great! Happy to see the K and O types available and in the data table. Well done! :)

**Comment by kapsakcj** [comment]
Thanks for the review and testing @emmadoughty and @frankambrosio3 

Since I merged #138 this morning, I had to resolve a number of merge conflicts for this PR before merging. I'm going to do some final testing in Terra and then merge once those run successfully

**Comment by kapsakcj** [comment]
Ran successfully in Terra post-resolving merge conflicts. I'm going to go ahead and merge!

**Pull request merged**